CsharpExtras/Map/Dictionary/Variant/IVariantDictionary.cs

```diff
@@ -0,0 +1,10 @@
+﻿using CsharpExtras.Map.Dictionary.Variant.Semi;
+using System.Collections.Generic;
+
+namespace CsharpExtras.Map.Dictionary.Variant
+{
+    //Non-mvp: Further refine the types sitting above this - convert some into the Map types

```

Comment:
This should probably be added as an issue as well as/instead of putting a comment in the code to improve visibility and better define what it means.

Reply:
Yeah it's not very critical at all so we can leave a non-mvp comment for now